docs/usage/configuration-options.md

```diff
@@ -1834,11 +1834,11 @@ For example, GitHub might automerge a Renovate branch even if it's behind the ba
 
 ## platformCommit
 
-Supports only GitHub App mode and not when using Personal Access Tokens.
+Only use this option if you run Renovate as a [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps).
+It does not apply when you use a Personal Access Token as credential.
```

Comment:
I like your suggestion, it's better than what we had! 😉 

I changed the text from passive voice to active voice:

```diff
- This option is possible to use only if Renovate is running as a GitHub App.
+ Only use this option if you run Renovate as a GitHub App.

- It does not apply when using a Personal Access Token for credentials.
+ It does not apply when you use a Personal Access Token as credential.
```